docs/learning/workshop_a_precision_converter_fpga_integration_journey/index.rst

```diff
@@ -0,0 +1,693 @@
+A Precision Converter FPGA Integration Journey
+==============================================
+
+Agenda
+~~~~~~
+
+1. Introduction
+2. SPI Engine Architecture
+3. What Is the Use Case?
+4. System Build
+5. System Evaluation
+6. Conclusions
+
+Introduction
+~~~~~~~~~~~~
+
+**Customer Journey**
+
+.. figure:: intro_customer_journey_1.png
+   :align: center
+
+Tools / Platforms are a customer choice, ADI ports to “current” development
+environment/kit.
+
+.. figure:: intro_customer_journey_2.png
+   :align: center
+
+Maintenance is an ADI burden. Customer don’t start their designs at the same
+time, and want to use the latest and greatest.
+
+**COS Reference Design “Donut Hole” Strategy**
+
+*Surround others processor, FPGA, micro ADI components*
+*Reference design creates stickiness*
+*Customer experiences low friction*
+*Leverage ecosystems*
+*Develop/participate in viable communities / ecosystems:*
+- Linux kernel (1.3 Billion users)
+- GitHub (40 Million users)
+- Python (100 Million users)
+- MATLAB (1 Million users)
```

Comment:
This is not rendered as separate lines, please review the page visually in the browser

Reply:
Done